docs-src/content/getting-started/streaming.md

```diff
@@ -69,4 +72,111 @@ func main() {
 }
 ```
 {{% /tab %}}
+
+{{% tab "Python" %}}
+``` python
+import cubic
+
+serverAddress = '127.0.0.1:2727'
+
+client = cubic.Client(serverAddress)
+
+# get list of available models
+modelResp = client.ListModels()
+for model in modelResp.models:
+	print("ID = {}, Name = {}".format(model.id, model.name))
+
+# use the first available model
+model = modelResp.models[0]
+
+cfg = cubic.RecognitionConfig(
+    model_id = model.id
+)
+
+# client.StreamingRecognize takes any binary
+# stream object that has a read(nBytes) method.
+# The method should return nBytes from the stream.
+
+# open audio file stream
+audio = open('test.raw', 'rb')
+
+# send streaming request to cubic and 
+# print out results as they come in
+for resp in client.StreamingRecognize(cfg, audio):
+    for result in resp.results:
+		if result.is_partial:
+			print("\r{0}".format(result.alternatives[0].transcript), end="")
+		else:
+			print("\r{0}".format(result.alternatives[0].transcript), end="\n")
+
+```
+{{% /tab %}}
+
+{{%/tabs %}}
+
+
+### Streaming from microphone
```

Comment:
Was this gonig to be a separate PR?

Reply:
Removed the microphone example; will put it up together with a parallel example for the go client in a separate PR